modules/apps/journal/journal-web/test/js/modals/ImportDataDefinitionModal.js

```diff
@@ -0,0 +1,212 @@
+
+		expect(title).toBeInTheDocument(true);
+	});
+
+	it('renders the info alert', () => {
+		const {queryByText} = container;
+
+		const infoAlert = queryByText(
+			'the-import-process-will-run-in-the-background-and-may-take-a-few-minutes'
+		);
+
+		expect(infoAlert).toBeInTheDocument(true);
+	});
+
+	it('renders name input', () => {
+		const {queryByText} = container;
+
+		const inputName = queryByText('name');
+
+		expect(inputName).toBeInTheDocument(true);
+	});
+
+	it('input name has correct name prop', () => {
+		const {getByLabelText} = container;
+
+		const inputName = getByLabelText('name');
+
+		expect(inputName).toHaveAttribute('name', 'testname');
+	});
+
+	it('renders json input', () => {
+		const {queryByText} = container;
+
+		const jsonInput = queryByText('json-file');
+
+		expect(jsonInput).toBeInTheDocument(true);
+	});
+
+	it('input json file has correct name prop', () => {
+		const inputs = document.getElementsByName('testjsonFile');
+
+		expect(!!inputs).toBe(true);
+	});
+
+	it('renders cancel button in footer', () => {
+		const {queryByText} = container;
+
+		const buttonCancel = queryByText('cancel');
+
+		expect(buttonCancel).toBeInTheDocument(true);
+	});
+
+	it('the cancel button closes modal', () => {
+		const {queryByText} = container;
+
+		const buttonCancel = queryByText('cancel');
+
+		userEvent.click(buttonCancel);
+
+		const body = document.getElementsByTagName('body').item(0);
+
+		expect(body).toHaveAttribute('class', '');
+	});
+
+	it('renders import button in footer', () => {
+		const {queryByText} = container;
+
+		const buttonImport = queryByText('import');
+
+		expect(buttonImport).toBeInTheDocument(true);
+	});
+
+	it(
+		'when exists a file selected,' +
+			' the file name appear in disabled input file',
+		() => {
+			const {queryByText} = container;
+
+			const inputFile = document.getElementsByName('testjsonFile')[0];
+			const inputFileName = document.getElementById('testjsonFile');
+
+			fireEvent.change(inputFile, {
+				target: {files: [file]},
+				value: 'C://downloads/Testing.lar',
+			});
+
+			expect(inputFileName).toHaveAttribute('value', 'Testing.lar');
+			expect(inputFileName).toHaveAttribute('disabled');
+
+			const clearButton = queryByText('clear');
+			fireEvent.click(clearButton);
+		}
+	);
+
+	it(
+		'the clear button dont appear,' +
+			' when not exists value in input file name',
+		() => {
+			const {queryByText} = container;
+
+			const clearButton = queryByText('clear');
+
+			expect(clearButton).not.toBeInTheDocument();
+		}
+	);
+
+	it('the clear button appear when exists value in input file name', () => {
+		const {queryByText} = container;
+
+		const inputFile = document.getElementsByName('testjsonFile')[0];
+		const inputFileName = document.getElementById('testjsonFile');
+
+		fireEvent.change(inputFile, {target: {files: [file]}});
+
+		const clearButton = queryByText('clear');
+
+		expect(inputFileName).toHaveAttribute('value', 'Testing.lar');
```

Comment:
use [toHaveValue()](https://github.com/testing-library/jest-dom#tohavevalue) instead of `toHaveAttribute()` to assert values:
```suggestion
		expect(inputFileName).toHaveValue('Testing.lar');
```